src/stub.js

```diff
@@ -0,0 +1,10 @@
+import m3u8 from './m3u8';
```

Comment:
stub.js will be removed on the final part of these pull requests and videojs-contrib-hls.js will be set as the main file again. stub.js is the main file for now because it allows us to keep all the unit tests passing by stubbing old functionality (adding m3u8 to video.js and Stream to videojs.Hls and more as the PRs go on).